.github/workflows/ci.yaml

```diff
@@ -7,11 +7,6 @@ name: ci
 jobs:
   build:
     runs-on: ubuntu-latest
-    container: gcr.io/gapic-images/googleapis-bazel:20210105
```

Comment:
Why was this removed?

Reply:
Sorry, forgot to explain in the description. Bazel version there is old, so builds fail with these changes. We encountered the same error on gax-java, and the J-Team had a discussion around this. `ubuntu-latest` comes with a recent Bazel, and we didn't find a good reason to use a custom image. We came to the agreement that we would just use `ubuntu-latest`.